src/main/resources/tags/info/bungee.tag

```diff
@@ -5,4 +5,10 @@ aliases: bungeecord, velocity
 
 The Geyser and Floodgate BungeeCord plugins only need to be installed on the BungeeCord proxy unless you intend to use the Floodgate API. That way, you need Floodgate on the "backend" servers too.
 Installing Floodgate on the backend servers also makes it so that Bedrock players don't have to switch backend servers for their skins to start displaying.
+
+To properly set up Floodgate on the proxy <-> Floodgate on the backend connection, do the following:
+1. Set up player-info-forwarding in the proxy configuration files.
+2. Replace the Floodgate key.pem file on the backend servers (the servers behind the proxy) with the key.pem file from the proxy.
+3. Enable "send-floodgate-data" in the Floodgate config.yml on the Proxy. Be careful - this requires ALL the backend servers to have floodgate installed.
+
```

Comment:
Can we just link to the wiki? I know people don't like following links but it has better resources on info forwarding for each proxy (isn't `player-info-forwarding` for velocity?)

Reply:
breh replied to wrong one
anyways, yes, I've added a new commit that mentions that this is velocity only vs bungee..which, ironically, the wiki doesn't do (no mention of velocity)
ig I'll make a wiki pr for that and just link to the wiki in this tag

Reply:
👍